src/conf.py

```diff
@@ -81,8 +81,8 @@
 # The encoding of source files.
 #source_encoding = 'utf-8-sig'
 
-# The master toctree document.
-master_doc = 'index'
+# The main toctree document.
```

Comment:
I suspect these were not supposed to be changed.

Reply:
I wasn't sure if those were our labels or used by the doc generation tool. I'll change back.

Reply:
It's a sphinx option. not ours.

Reply:
Changed back.